x-pack/plugins/security/server/authentication/index.ts

```diff
@@ -65,18 +65,42 @@ export async function setupAuthentication({
       .callAsCurrentUser('shield.authenticate')) as AuthenticatedUser;
   };
 
+  const isValid = (sessionValue: ProviderSession) => {
+    // ensure that this cookie was created with the current Kibana configuration
+    if (sessionValue.path !== (http.basePath.serverBasePath || '/')) {
```

Comment:
When upgrading from a previous version of Kibana to this version, this will cause all existing sessions to be invalided. Perhaps we should ignore the path check if `sessionValue.path === undefined`?

Reply:
Sure, the cookie would get rewritten anyway so there's no real reason to invalidate it.